src/monodroid/jni/util.h

```diff
@@ -15,9 +15,20 @@
 #define MONODROID_PATH_SEPARATOR "/"
 #endif
 
+#if WINDOWS
+typedef struct _stat monodroid_stat_t;
+#define monodroid_dir_t _WDIR
```

Comment:
The compiler for some reason didn't like that and was considering the types to be different: 

`expected 'struct _WDIR *' but argument is of type 'struct monodroid_dir_t *'`